_topic_map.yml

```
@@ -337,6 +335,15 @@ Topics:
   File: retrieving-and-analyzing-the-collector-logs-and-pod-status
 - Name: Commonly occurring error conditions
   File: commonly-occurring-error-conditions
+---  
+Name: Troubleshooting Central
```

It is probably understood well enough in context, but at a glance this might appear / be misread as a "central hub" for Troubleshooting in general. Suggest adding a bit more description on what "Central" is for the title:
```suggestion
Name: Troubleshooting the Central database
```

---

Thank you for the suggestion. However, I would prefer to leave it as it is, since Central is one of the components we have in the Central services.
https://docs.openshift.com/acs/4.3/architecture/acs-architecture.html#:~:text=DB%20and%20Scanner.-,Central,-%3A%20Central%20is%20the